leapp/reporting/__init__.py

```diff
@@ -125,6 +126,23 @@ def __init__(self, value=None):
         self._value = value
 
 
+class Key(BasePrimitive):
+    """Stable identifier for report entries"""
+    name = 'key'
+
+    def __init__(self, entries):
+
+        def _find_entry(cls):
+            entry = next((e for e in entries if isinstance(e, cls)), None)
```

Comment:
not principal:
in python usually preferred https://en.wikipedia.org/wiki/EAFP

Reply:
I'd keep it as is, it's shorter